pyproject.toml

```diff
@@ -62,11 +62,18 @@ all = [
     "html5lib>=1.0",
     "inquirer>=3.3.0",
     "jaro-winkler>=2.0.3",
+    "opentelemetry-api>=1.34.1",
+    "opentelemetry-distro[otlp]",
+    "opentelemetry-instrumentation",
+    "opentelemetry-instrumentation-httpx",
+    "opentelemetry-sdk>=1.34.1",
+    "opentelemetry-semantic-conventions",
```

Comment:
Don't we want to set minimum versions to other opentelemetry packages (and wrapt) as well?